contracts/ethereum/tools/lib/utils/utils.ts

```diff
@@ -8,17 +9,34 @@ export default class Utils {
     this.web3 = web3;
   }
 
-  public async createSignature(gene: any): Promise<string> {
+  private async loadOwner(): Promise<string> {
     const defaultOwner = await this.web3.eth.accounts.wallet[0].address;
-    if (defaultOwner === undefined){
+    if (defaultOwner === undefined) {
       throw new Error('at least one wallet must exist');
     }
-    const hash = await this.web3.utils.soliditySha3(gene, defaultOwner);
+
+    return defaultOwner
+  }
+
+  public async createGeneSignature(gene: Mixed): Promise<string> {
+    return await this.createSignature(gene, await this.loadOwner())
```

Comment:
Why did `this.loadOwner()` used as a parameter??

In `createSignature()`, it was already signed with `deafaultOwner`.

Reply:
(gene,owner) is the data to be signed!!